### PR TITLE
Updating UniFi Documentation

### PR DESCRIPTION
**Pull request by angrycuban13**
# Pull request

**Purpose**
Some ELI5 for why we update the Controller's IP.

### CODE REVIEW

**Review by jodfie — CHANGES_REQUESTED** [review]
Typo Suck instead of Stuck

**Comment by angrycuban13** [comment]
> Typo Suck instead of Stuck

Fixed on 899bde0

**Review by nemchik — APPROVED** [review]

**Comment by nemchik** [comment]
@jodfie can you review again and approve/merge when ready?

**Review by jodfie — APPROVED** [review]

**Comment by jodfie** [comment]
> @jodfie can you review again and approve/merge when ready?

Done! Sorry it didn't know it locks it to me having to go back and approve!
